.github/infra/cloudbuild-configure-triggers.yaml

```diff
@@ -17,12 +17,17 @@
 
 steps:
 - name: 'google/cloud-sdk:slim'
-  id: Configure build triggers deployment
+  id: Configure build triggers
   entrypoint: /bin/bash
   args:
   - -c
   - |
     set -o errexit
     set -o xtrace
-    gcloud beta deployment-manager deployments create cloud-build-triggers --config .github/infra/build-triggers.yaml || \
-      gcloud beta deployment-manager deployments update cloud-build-triggers --config .github/infra/build-triggers.yaml
+    for f in .github/infra/triggers/*.yaml
```

Comment:
This won't actually delete a trigger if the file is deleted. Is that expected?

Reply:
Good catch. I'm comfortable leaving deletion as a manual process for now, mostly because I think it would be really hard to automate well (how do you tell when it's appropriate to delete a trigger that is not represented by a YAML file in this directory?). WDYT?

Reply:
We could probably do something with tags where we set a specific tag on triggers created by files in the directory and then delete all triggers with that tag before importing them.

Does this script have to be manually run when modifying or adding a file or is it somehow automatically run?

If it's automatically run, I think there's a greater expectation that the triggers stay in sync with the ones in the directory. If it's manually run, I think there's less of an expectation.

Reply:
I totally agree. For context, Deployment Manager addresses all of these concerns, however it currently has an open bug that prevents us from using it to configure Cloud Build triggers (see #98). I have added a TODO to this file to reflect that we want to switch back to Deployment Manager once the bug is fixed; until then, I'm inclined to live with this as-is because to make it more robust would just be duplicating the functionality of Deployment Manager.